project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "1.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.9.1")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.9.5")
```

Comment:
why not 3.0.5?

Reply:
I did not want to do a major release update. However, now looking at [here](https://central.sonatype.com/artifact/com.typesafe.play/sbt-plugin/versions) I cant find any release after 2.9.5. Can you tell me where you saw that newest version?

Reply:
Make sense, let's make it 3.0.5. Orchard did it already and it's almost identical to 2.9.x except it uses pekko instead of akka, which will have much better security updates and better license. You might need to change a few imports from `akka` to `pekko` and rename `Akka` to `Pekko` here and there to make it work. But that's all the changes needed.

Here is the link for the new version: https://central.sonatype.com/artifact/org.playframework/sbt-plugin/overview

Reply:
@realstraw Do you mind merging this PR and I work on that upgrade in a different PR as it would be more involved and requires a lot more testing/validation?